gno.land/cmd/gnoweb/pages/HOME.md

```diff
@@ -1,50 +1,77 @@
+[Explore Github repo](https://github.com/gnolang)
+
+</div>
+
+<div class="jumbotron">
+
+### Game of Realms
+
+The first high-stakes contest will see participants compete for tiered membership to co-own the Gno.land blockchain. A series of complex technical and non-technical tasks will challenge contributors to create innovative patterns that push the chain to new limits. Start building the foundation for tomorrow through key smart contracts and other contributions that change our understanding of the world.
+
+<div class="stack">
+
+[Join Game Of Realms](/game-of-realms)
+
+</div>
+</div>
+
+<div class="columns-3">
+<div>
+
+### Learn about Gno.land
+
+- [About](/about)
+- [Subscribe](/)
+- [Tokenomics (soon)](#)
 - [Blogs](/r/gnoland/blog)
-- [Install `gnokey`](https://github.com/gnolang/gno/tree/master/gno.land/cmd/gnokey)
-- [Acquire testnet tokens](/faucet)
-- [Game of Realms](/game-of-realms) - An open worldwide competition for developers to build the best Gnolang smart-contracts.
-
-# Explore new packages.
-
-- r/gnoland
-  - [/r/gnoland/blog](/r/gnoland/blog)
-  - [/r/gnoland/faucet](/r/gnoland/faucet)
-- r/system
-  - [/r/system/names](/r/system/names)
-  - [/r/system/rewards](/r/system/rewards)
-  - [/r/system/validators](/r/system/validators)
-- r/demo
-  - [/r/demo/banktest](/r/demo/banktest)
-  - [/r/demo/boards](/r/demo/boards)
-  - [/r/demo/foo20](/r/demo/foo20)
-  - [/r/demo/nft](/r/demo/nft)
-  - [/r/demo/types](/r/demo/types)
-  - [/r/demo/users](/r/demo/users)
-  - [/r/demo/groups](/r/demo/groups)
-- p/demo
-  - [/p/demo/avl](/p/demo/avl)
-  - [/p/demo/blog](/p/demo/blog)
-  - [/p/demo/flow](/p/demo/flow)
-  - [/p/demo/gnode](/p/demo/gnode)
-  - [/p/demo/grc/exts](/p/demo/grc/exts)
-  - [/p/demo/grc/grc20](/p/demo/grc/grc20)
-  - [/p/demo/grc/grc721](/p/demo/grc/grc721)
-
-# Other Testnets
-
-- **[staging.gno.land](https://staging.gno.land) (wiped every commit to master)**
-- _[test3.gno.land](https://test3.gno.land) (latest)_
-- _[test2.gno.land](https://test2.gno.land) (archive)_
-- _[test1.gno.land](https://test1.gno.land) (archive)_
-
-**This is a testnet.**
-Package names are not guaranteed to be available for production.
-
-# Social
-
-Check out our [community projects](https://github.com/gnolang/awesome-gno).
-
-Official channel: [Discord](https://discord.gg/S8nKUqwkPn)<br />
-Other channels: [Telegram](https://t.me/gnoland) [Twitter](https://twitter.com/_gnoland) [Youtube](https://www.youtube.com/@_gnoland)
+- [Events](/events)
+- [Partners (soon)](#)
+
+</div>
+
+<div>
+
+### Build with Gnolang
+
+- [Gno dev with CLI](/)
+- [Gno IDE (soon)](#)
+- [Test in the browser (soon)](#)
+- [Docs/ Tutorials](https://github.com/gnolang)
+- [Workshops/ Events/ Office hours](/events)
+- [Getting started video (soon)](#)
+
+</div>
+<div>
+
+### Explore the universe
+
+- [Discover demo packages](https://github.com/gnolang/gno/tree/master/examples)
+- [Install Gno Key instructions](/r/demo/boards:testboard/5)
```

Comment:
The instructions in the post are not up-to-date , I will create a separate issue to fix them.

Reply:
https://github.com/gnolang/gno/issues/865